pombola/feedback/views.py

```diff
@@ -2,6 +2,8 @@
 from django.template import RequestContext
 from django.views.decorators.csrf import csrf_protect
 
+from pombola.search.recaptcha import recaptcha_client
```

Comment:
mmm it looks like recaptcha_client is now more general than the `search` package, it isn't actually specific to search, is it? Let's move it to a move appropriate place.

Reply:
Looks like you moved the test suite to the new location but didn't remove it from the old? hence it showing up as a new module and the old version failing /app/pombola/search/tests/recaptcha.py